draft-ietf-quic-tls.md

```diff
@@ -1570,12 +1564,23 @@ Note:
   packet protection keys.  To avoid exhaustion of both sets of keys, it might be
   necessary to initiate two key updates in succession.
 
-For AEAD_AES_128_GCM, AEAD_AES_256_GCM, and AEAD_CHACHA20_POLY1305, the limit on
-the number of packets that fail authentication is 2^36.  Note that the analysis
-in {{AEBounds}} supports a higher limit for the AEAD_AES_128_GCM and
-AEAD_AES_256_GCM, but this specification recommends a lower limit.  For
-AEAD_AES_128_CCM, the limit on the number of packets that fail authentication
-is 2^23.5; see {{ccm-bounds}}.
+A connection MUST be closed if the number of encrypted packets exceeds the
```

Comment:
I missed this last time around.

I think that this is unrealistic.  People will make new connections, which have the same properties as new keys.  So we should just recommend key updates.

Reply:
Hmm, fair point. They are functionally the same thing since we're ignoring the number of users. That said, doesn't it seem inconsistent to drop connections in the case of integrity, but then re-key for confidentiality?